modules/pubsub/src/main/scala/com/snowplowanalytics/snowplow/enrich/pubsub/Source.scala

```diff
@@ -44,12 +47,24 @@ object Source {
   ): Stream[F, ConsumerRecord[F, Array[Byte]]] = {
     val onFailedTerminate: Throwable => F[Unit] =
       e => Sync[F].delay(System.err.println(s"Cannot terminate ${e.getMessage}"))
+
+    val flowControlSettings: FlowControlSettings =
+      FlowControlSettings
+        .newBuilder()
+        .setMaxOutstandingElementCount(input.maxQueueSize)
+        .setMaxOutstandingRequestBytes(input.maxRequestBytes)
+        .setLimitExceededBehavior(LimitExceededBehavior.Block)
+        .build()
+
     val pubSubConfig =
       PubsubGoogleConsumerConfig(
         onFailedTerminate = onFailedTerminate,
         parallelPullCount = input.parallelPullCount,
-        maxQueueSize = input.maxQueueSize
+        maxQueueSize = input.maxQueueSize,
```

Comment:
Do we still need this line?

Reply:
yes, to set size of fs2-pubsub's internal queue

Reply:
Ah yes I now get the difference, thanks!